### PR TITLE
s/cosign/sigstore/

### PR DESCRIPTION
**Pull request by mtrmac**
… per recent discussions.

Note for anyone experimenting with this: this PR changes the on-disk binary format of the signature (which was not yet in a c/image release).

<s>Not tested in practice so far to ensure this maintains interoperability.</s>

### CODE REVIEW

File: docs/containers-registries.d.5.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ described above.  The configuration section is a YAML mapping, with the followin
    This key is optional; if it is missing, no signature storage is defined (no signatures
    are download along with images, adding new signatures is possible only if `sigstore-staging` is defined).
 
-- `use-cosign-attachments` specifies whether Cosign image attachments (signatures, attestations and the like) are going to be read/written along with the image.
+- `use-sigstore-attachments` specifies whether sigstore image attachments (signatures, attestations and the like) are going to be read/written along with the image.
```
Comment (mtrmac) [inline docs/containers-registries.d.5.md:87]: @rhatdan @vrothberg If this should be `use-attachments` or something else, please speak up.

(The `s/sigstore/lookaside/` changes will be a separate PR.)

File: docs/containers-policy.json.5.md [hunk 2]
```diff
@@ -237,19 +237,19 @@ used with `exactReference` or `exactRepository`.
 <!-- ### `signedBaseLayer` -->
 
 
-### `cosignSigned`
+### `sigstoreSigned`
```
Comment (mtrmac) [inline docs/containers-policy.json.5.md:240]: @rhatdan @vrothberg PTAL

**Comment by mtrmac** [comment]
Manually smoke-tested with https://github.com/containers/skopeo/pull/1701 : Skopeo accepts Cosign-created signatures and vice versa.

**Comment by rhatdan** [comment]
LGTM

**Review by vrothberg — COMMENTED** [review]
LGTM
Freel free to merge after rebasing
